### PR TITLE
Fix Application & Arguments can't be save from Preferences

### PR DESCRIPTION
**Pull request by linying1991**
[fix the issue 165](https://github.com/SketchUp/testup-2/issues/165)

image

### CODE REVIEW

File: dialogs/preferences.ts [hunk 1]
```diff
@@ -98,6 +98,12 @@ window.app = new Vue({
     addPath() {
       sketchup.addPath();
     },
+    changeEditorExecutable: function(ev) {
```
Comment (thomthom) [inline dialogs/preferences.ts]: Coding style: Avoid abbreviations, prefer `event` over `e` or `ev`.

File: src/testup/ui/html/preferences.html [hunk 2]
```diff
@@ -58,11 +58,11 @@
       <su-group title="Code Editor" id="tuCodeEditor">
 
         <div class="tu-input-group">
-          <su-input label="Application:" v-model="config.editor.executable"></su-input>
+          <su-input label="Application:" v-on:keyup.native="changeEditorExecutable" v-model="config.editor.executable"></su-input>
```
Comment (thomthom) [inline src/testup/ui/html/preferences.html]: Hmm... this doesn't look right. `v-model` should be taking care of this - bending to `config.editor.executable` and updating it. `keyup` would not update if something was copied and pasted - while `v-model` should. We'd better figure out why `v-model` isn't properly updating.

File: src/testup/ui/preferences.rb [hunk 3]
```diff
@@ -94,7 +94,7 @@ def event_save_config(config)
       Log.info "event_save_config(...)"
       Log.debug config.inspect
       # Save editor config:
-      application = config['editor']['application']
+      application = config['editor']['executable']
```
Comment (thomthom) [inline src/testup/ui/preferences.rb]: Oh - isn't this the core of this but? Using the wrong key? Was the `onkey` events `preferences.ts` necessary?

Comment (linying1991) [reply src/testup/ui/preferences.rb]: Yes, 'application' is a wrong key.

File: src/testup/ui/html/preferences.html (hunk 2 above)
Comment (linying1991) [reply src/testup/ui/html/preferences.html]: I am very sorry, when I modified this bug, I only considered solving the problem and forgot that this is open source collaboration. Thank you for your reminder.

Comment (thomthom) [reply src/testup/ui/html/preferences.html]: No problem - I know what its like when you just need to get something blocking you fixed.

**Comment by thomthom** [comment]
Closing this because I fixed it via #174

**Pull request closed without merging**